realm/realm-library/src/objectServer/java/io/realm/internal/network/OkHttpNetworkTransport.java

```diff
@@ -90,7 +99,10 @@ private synchronized OkHttpClient getClient(long timeoutMs) {
                                     // We only expect request context to be JSON.
                                     Buffer buffer = new Buffer();
                                     request.body().writeTo(buffer);
-                                    sb.append(buffer.readString(UTF8));
+
```

Comment:
Maybe we should unwrap this into an actual `LoggingInterceptor` that could be passed into the `OkHttpNetworkTransport`.

Reply:
Sounds like a good idea.